microprofile-fault-tolerance/README.adoc

```diff
@@ -0,0 +1,433 @@
+////
+This guide is _ported_ from the main Quarkus repository
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+include::../shared-doc/attributes.adoc[]
+
+= microprofile-fault-tolerance: MicroProfile Fault Tolerance
+:author: Tomas Hofman, Radoslav Husar
+:level: Intermediate
+:technologies: MicroProfile, Fault Tolerance
+
+[abstract]
+The `microprofile-fault-tolerance` quickstart demonstrates how to use Eclipse MicroProfile Fault Tolerance in {productName}.
+
+One of the challenges brought by the distributed nature of microservices is that communication with external systems is
+inherently unreliable. This increases demand on resiliency of applications. To simplify making more resilient
+applications, {productName} contains an implementation of the MicroProfile Fault Tolerance specification.
+
+In this guide, we demonstrate usage of MicroProfile Fault Tolerance annotations such as `@Timeout`, `@Fallback`,
+`@Retry` and `@CircuitBreaker`. The specification also introduces `@Bulkhead` and `@Asynchronous` interceptor bindings not
```

Comment:
Why are they not covered here?

Reply:
This is whatever Quarkus has, expanding this further I don't consider useful time investment, but contributions are welcome!

Reply:
PS: contributions are welcome. These documents are not immutable ;-)